src/views/registration-js/attendees/content.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * This template renders the registration/purchase attendee fields
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/registration-js/attendees/content.php
+ *
+ * @since TBD
+ *
+ * @version TBD
+ *
+ */
+if (
+	! class_exists( 'Tribe__Tickets_Plus__Meta' )
+	|| ! class_exists( 'Tribe__Tickets_Plus__Meta__Storage' )
+) {
+	return;
+}
+
+$storage = new Tribe__Tickets_Plus__Meta__Storage();
+$meta    = tribe( 'tickets-plus.main' )->meta();
+?>
+
+<?php foreach ( $tickets as $ticket ) : ?>
+		<?php
+		// Only include tickets with meta
+		$has_meta = get_post_meta( $ticket['id'], '_tribe_tickets_meta_enabled', true );
+
+		if ( empty( $has_meta ) || ! tribe_is_truthy( $has_meta ) ) {
+			continue;
+		}
+		?>
+		<script type="text/html" id="tmpl-tribe-registration--<?php echo esc_attr( $ticket['id'] ); ?>">
+			<?php
+
+			$ticket_qty = $ticket['qty'];
+			$post           = get_post( $ticket['id'] );
+			?>
+			<?php // go through each attendee ?>
+			<?php while ( 0 < $ticket_qty ) : ?>
+				<?php
+					/**
+					* @var Tribe__Tickets_Plus__Meta $meta
+					*/
+					$fields     = $meta->get_meta_fields_by_ticket( $post->ID );
```

Comment:
Any reason this is done inside the while/foreach loop instead of up above where we setup `$meta` and `$storage` vars?

Reply:
Nope - fixing.